### ISSUE TITLE
removing empty imports

### ISSUE THREAD

**Issue by hntrl** [coderaiser/putout#5]
Running remove-unused-variables on
`import { Foo } from 'bar';`
produces
`import 'bar';`

Logically, i'd think this is expected, but I still need to get rid of these imports somehow.
Writing a plugin seems like the only option, but I have no idea how to "connect" these plugins together to get the result i'm looking for

```js
// use remove-unused-vars
import { Foo } from 'bar';
import 'foo';
```
```js
// use remove-unused-imports
import 'bar';  // <-- fix this
import 'foo'
```

**Comment by coderaiser** [comment]
I can't reproduce this is `cli`. Are you talking about `API`?

**Comment by coderaiser** [comment]
Landed in [v4.32.1](https://github.com/coderaiser/putout/releases/tag/v4.32.1) 🎉, thank you 🙂.
